src/Pages/Home.js

```diff
@@ -6,7 +6,21 @@ function Home(){
         <div className="Home">
             {/* Hero section container  */}
             <div className="Hero_section">
-                <img className="Hero_image" src={require('../Assets/HeroImg.jpg')} />
+            <img className="Hero_image" src= {require('../Assets/HeroImg.jpg')} />
+                <div className="Hero_overlay">
+                    <p className="style_2">Hello, my name is</p>
+                    <h1 className="style_1">FRANKIE <span id="style_1">HRNDZ</span></h1>
```

Comment:
Why did you choose to add a span for “HRNDZ” if its on the same line as “FRANKIE” and has the same styling?

Reply:
It uses some of the same styling in terms of font family and size but the "HRNDZ" is styled differently. I  uses a keystroke styling to make "HRNDZ" appear transparent with a only the border of the text being displayed.